package.json

```diff
@@ -24,11 +24,12 @@
   "devDependencies": {
     "chai": "^4.2.0",
     "chai-http": "^4.3.0",
-    "mocha": "^7.0.1",
-    "nyc": "^15.0.0",
+    "coveralls": "^3.0.9",
     "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.0.0",
     "eslint-plugin-import": "^2.20.1",
-    "eslint-config-airbnb-base": "^14.0.0"
+    "mocha": "^7.0.1",
+    "nyc": "^15.0.0"
```

Comment:
where have we used this?

Reply:
Mnickii, I pushed a script for running coverage with a mocha reporter recently.